modules/Attendance/report_graph_byType.php

```diff
@@ -110,14 +110,22 @@ function getDateRange($guid, $connection2, $first, $last, $step = '+1 day', $out
         $row->addDate('dateEnd')->setValue(dateConvertBack($guid, $dateEnd))->isRequired();
 
     $typeOptions = array_column($attendance->getAttendanceTypes(), 'name');
+    $arrayTypeOptions = array();
+    foreach($typeOptions as $types){
```

Comment:
Another option here is to use `$typeOptions = array_map('__', $typeOptions);`. It's useful when you have one function you'd like to apply to the whole array.